scenes/test_object.txt

```diff
@@ -0,0 +1,45 @@
+*label entry
+*params module log_buffer
+*temp test_count 0
+*temp tests_passed 0
+
+*gosub test_start "get unmodified data"
+*gosub_scene cslib_object get "testobj" 1
+*gosub test_assert (testobj_testfield1 = "Calvin") "=> Expected - Calvin[n/]=> Received - ${testobj_testfield1}[n/]"
+*gosub test_finish
+
+*gosub test_start "set (modify) data"
+*gosub_scene cslib_object set "testobj" 2 "testfield1" "Locke"
+*gosub test_assert (testobj_2_testfield1 = "Locke") "=> Expected - Locke[n/]=> Received - ${testobj_2_testfield1}[n/]"
+*gosub test_finish
+
+*gosub test_start "get modified data"
+*gosub_scene cslib_object get "testobj" 2
+*gosub test_assert (testobj_testfield1 = "Locke") "=> Expected - Locke[n/]=> Received - ${testobj_testfield1}[n/]"
```

Comment:
It's better to avoid having tests depending on one another. Things will get messy if one of them is moved around, changed or removed.

A test can have multiple `test_assert` statements, so I suggest just moving this one into the test above (and renaming the test to `set and get (modified) data` accordingly.

Reply:
OK